pkg/sql/logictest/testdata/logic_test/tenant

```diff
@@ -45,3 +45,49 @@ user testuser
 
 statement error only users with the admin role are allowed to create tenant
 CREATE TENANT "not-allowed"
+
+subtest drop_tenant
+user root
+
+statement error destroying tenant: tenant "dne" does not exist
+DROP TENANT dne
```

Comment:
feel free to ignore but maybe a test with a tenant name in ""?

Reply:
doneburger